### PR TITLE
Feature/data download

### PR DESCRIPTION
**Pull request by qpfiffer**
Adds download buttons to all of the charts so that you can download them, bro.

### CODE REVIEW

File: server/apps/survey/management/commands/test_data.py [hunk 1]
```diff
@@ -62,17 +63,18 @@ def handle(self, *args, **options):
             slug='survey-date', survey=survey)
         users = []
         for i in range(7):
-            user = User.objects.get(username='user{0}'.format(i))
-            if user:
+            try:
+                user = User.objects.get(username='user{0}'.format(i))
                 users.append(user)
-            else:
+            except User.DoesNotExist:
```
Comment (wraithan) [inline server/apps/survey/management/commands/test_data.py:68]: Whoops, nice catch.

File: server/apps/survey/management/commands/test_data.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 from django.contrib.auth.models import User
 from django.core.management.base import BaseCommand
+from django.utils.timezone import utc
```
Comment (wraithan) [inline server/apps/survey/management/commands/test_data.py]: Not used in this file.

File: server/apps/reports/views.py [hunk 3]
```diff
@@ -290,7 +290,7 @@ def surveyor_stats_raw_data_csv(request, survey_slug):
     writer = csv.writer(response)
     writer.writerow(('Surveyor', 'market', 'timestamp', 'status'))
     for row in res:
-        writer.writerow((row.surveyor.get_full_name(), row.survey_site,
-                         row.ts, row.review_status))
+        writer.writerow((row.surveyor.get_full_name() if row.surveyor else '',
```
Comment (wraithan) [inline server/apps/reports/views.py:293]: Forgot this could be `None`.

File: server/apps/survey/management/commands/test_data.py (hunk 2 above)
Comment (qpfiffer) [inline server/apps/survey/management/commands/test_data.py]: Ah, you're right. This was a holdover from trying to fix the Datetime warnings when test_data runs. Will remove.

**Comment by qpfiffer** [comment]
Going to merge this in since @wraithan has looked at it.

**Pull request merged**